revolv/static/mixins.scss

```diff
@@ -31,3 +31,169 @@
     -webkit-transform: $transforms;
     transform: $transforms;
 }
+
+.revolv-reveal-modal-table {
```

Comment:
That makes sense actually. A lot of the same code is copied in modal.scss already - do you just want me to delete duplicate code from mixins.scss?